lib/html-processor.js

```diff
@@ -309,6 +309,16 @@ class HtmlProcessor {
 
         return HtmlProcessor.normalizeOutput(html, $);
     }
+
+    static propagateDirProperty(selector, html) {
```

Comment:
OK, interesting approach... I hope it's enough to go just one level down. Is it already online to test or do I need to install a local server?

Reply:
Not out yet - I still need to test it out manually, but the integration + unit tests suggest this should work.

I used the BBC article you provided to test that the property would get sent down to all the children (not just one level). So I'm optimistic!